sqlglot/dialects/dialect.py

```diff
@@ -46,6 +46,15 @@ class Dialects(str, Enum):
     Doris = "doris"
 
 
+class NormalizationStrategy(str, Enum):
+    """Specifies the strategy according to which identifiers should be normalized."""
+
+    LOWERCASE = "lowercase"  # Unquoted identifiers are lowercased
```

Comment:
why not use autoname like tokentype

Reply:
sure, should we do that for `Dialects` as well or is there a reason we've hardcoded the enum values there?

Reply:
do they map one to one? if so then yea

Reply:
refactored the normalization strategy but not the dialects because `DIALECT` maps to `""` so it's not 1-1